install/operator/pkg/cmd/internal/olm/olm.go

```diff
@@ -50,7 +50,7 @@ func New(parent *internal.Options) *cobra.Command {
 }
 
 func (o *Olm) generate() (err error) {
-	conf, err := configuration.GetProperties(context.TODO(), configuration.TemplateConfig, o.ClientTools(), &v1beta1.Syndesis{})
+	conf, err := configuration.GetProperties(context.TODO(), configuration.TemplateConfig, nil, &v1beta1.Syndesis{})
```

Comment:
Why is this necessary?

Is it to stop the ApiCapabilities being analysed?

Reply:
I don't need to any client in the context of the `olm` command. I think the best way to express this in a declarative way is setting the clientool to nil. The change below is to make this code work.